server-common/src/test/java/org/apache/kafka/server/common/MetadataVersionTest.java

```diff
@@ -246,8 +243,6 @@ public void testShortVersion() {
         assertEquals("3.7", IBP_3_7_IV2.shortVersion());
         assertEquals("3.7", IBP_3_7_IV3.shortVersion());
         assertEquals("3.7", IBP_3_7_IV4.shortVersion());
-        assertEquals("3.8", IBP_3_8_IV0.shortVersion());
```

Comment:
I guess that we can keep this `assertEquals("3.8", IBP_3_8_IV0.shortVersion());`?

Reply:
Hmm. It's strange that this was missing before. I can add it back.